src/nspawn/nspawn-mount.h

```diff
@@ -23,6 +23,14 @@
 
 #include "cgroup-util.h"
 
+typedef enum MountSettingsMask {
+        MOUNT_FATAL             = 1 << 0,
+        MOUNT_IN_USERNS         = 1 << 1,
+        MOUNT_USE_USERNS        = 1 << 3,
+        MOUNT_USE_NETNS         = 1 << 2,
+        MOUNT_RO                = 1 << 4,
```

Comment:
Hmm, the RO flag is a bit misleadingly named, no? after all it doesn't control read-only vs. writable. Instead it is simply a mask for the actual RO bit used. As such, it should probably named MASK_RO instead of just RO, no?

(or did I misunderstand this?)